src/frontend/screens/Settings/CreateOrJoinProject/JoinExistingProject.tsx

```diff
@@ -27,34 +29,75 @@ export const JoinExistingProject = ({
   const {formatMessage} = useIntl();
   return (
     <View style={styles.container}>
-      <View>
-        <Text style={[styles.text, {fontSize: 20}]}>
+      <View style={styles.topSection}>
+        <View style={styles.helpIconContainer}>
+          <HeaderText variant="header1" style={styles.helpIconText}>
+            ?
+          </HeaderText>
+        </View>
+
+        <HeaderText variant="header1" style={styles.heading}>
           {formatMessage(m.howTo)}
-        </Text>
-        <Text style={[styles.text, {marginTop: 20}]}>
+        </HeaderText>
+
+        <BodyText style={styles.instructions}>
           {formatMessage(m.instructions)}
-        </Text>
+        </BodyText>
+      </View>
+
+      <View style={styles.footer}>
+        <Button
+          fullWidth
+          variant="outlined"
+          style={{
+            borderColor: BLUE_GREY,
+          }}
+          color="ComapeoBlue"
+          onPress={() => {
+            navigation.goBack();
+          }}>
+          {formatMessage(m.goBack)}
+        </Button>
       </View>
-      <Button
-        fullWidth
-        variant="outlined"
-        onPress={() => {
-          navigation.goBack();
-        }}>
-        {formatMessage(m.goBack)}
-      </Button>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    padding: 20,
+    flex: 1,
+    paddingHorizontal: 40,
+  },
+  topSection: {
     paddingTop: 80,
-    justifyContent: 'space-between',
-    height: '100%',
+    alignItems: 'center',
+    gap: 20,
+  },
+  helpIconContainer: {
+    height: 80,
+    width: 80,
+    borderRadius: 40,
+    borderWidth: 1,
+    borderColor: NEW_DARK_GREY,
+    justifyContent: 'center',
+    alignItems: 'center',
   },
-  text: {
+  helpIconText: {
+    color: NEW_DARK_GREY,
+  },
+  heading: {
     textAlign: 'center',
+    color: DARK_GREY,
+  },
+  instructions: {
+    textAlign: 'center',
+    color: DARK_GREY,
+    lineHeight: 24,
```

Comment:
out of curiosity: wonder if this line height should be shared by all usages of `BodyText`?